server/mailet/mailets/pom.xml

```diff
@@ -133,6 +133,19 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>james-server-lifecycle-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>james-server-protocols-library</artifactId>
```

Comment:
it probably shows that tests are badly located.

Reply:
Moving the MockSpamd\* test helper stuff to james-testing stuff might solve this PB.

Would you agree with it @mbaechler ?